src/test/java/io/vertx/redis/client/test/RedisOptionsTest.java

```diff
@@ -1,6 +1,7 @@
 package io.vertx.redis.client.test;
 
 import io.vertx.redis.client.RedisOptions;
+import io.vertx.redis.client.RedisRole;
```

Comment:
Seems useless?

Reply:
From a previous iteration of the test where I was setting options. Removed. Thanks!